be/src/vec/columns/column.h

```diff
@@ -183,6 +183,11 @@ class IColumn : public COW<IColumn> {
     /// All data will be inserted as single element
     virtual void insert_data(const char* pos, size_t length) = 0;
 
+    // just used for ColumnDecimal to be compatible with decimalv2
+    virtual void insert_many_decimalv2_data(const char* pos, size_t num) {
```

Comment:
`insert_many_decimalv2_data` is not a common function, it is better not add it to IColumn.

While using, you could cast IColumn to ColumnDecimal, and call the function.

Reply:
This function will be called in bitshuffle_page.h. ColumnDecimal<T> is template class，but we don't know the real type of T in bitshuffle_page.h. so we can't  use static_cast here.

There are several solutions：
1. use dynamic_cast
2. change insert_many_decimalv2_data to a template function

I don't think they are better than the current implementation.